cassandra/cqlengine/management.py

```diff
@@ -121,7 +121,9 @@ def _get_index_name_by_column(table, column_name):
 
 def sync_table(model, keyspaces=None):
     """
-    Inspects the model and creates / updates the corresponding table and columns for all keyspaces.
+    Inspects the model and creates / updates the corresponding table and columns.
+
+    If `keyspaces` is specified, the table will be synched for all keyspaces. Note that the `Model.__keyspace__` is ignored in that case.
```

Comment:
I suggest "all *specified* keyspaces". The way its worded now makes it sound like it will put the table in every keyspace on the cluster. :)